### DISCUSSION TITLE
Adding TypeIt as shortcode

### DISCUSSION THREAD

**Discussion by mztiq** [nunocoracao/blowfish#334] [Ideas]
I saw someone using [TypeIt](https://www.typeitjs.com/) in their blog (see [Link](https://loisvelasco.is-a.dev/)) by customizing a Partial.
Sadly I couldn't manage to integrate TypeIt the way he did but I know there are Hugo themes that integrated TypeIt as Shortcode (Like [LoveIt](https://hugoloveit.com/theme-documentation-typeit-shortcode/)) so I was wondering if this could be added as a feature to Blowfish too.

Thanks for the great theme btw, Blowfish being free seems too good to be true 😉.

**Comment by nunocoracao** [comment]
@mztiq approved. can you open up the ticket for it?

**Reply by mztiq** [reply]
Thanks, I just did.
